os.h

```diff
@@ -0,0 +1,16 @@
+// Legacy BASIC
+// Utility types and functions.
+// Copyright (c) 2021-2 Nigel Perks
+
+#ifndef OS_H
+#define OS_H
+
+#ifdef LINUX
+#define __STRICMP strcasecmp
+#define __STRNICMP strncasecmp
```

Comment:
I believe identifiers beginning with two underscores (or an underscore and capital letter) are reserved in ISO standard C. Can we change it to just STRICMP ?